how-to/customize-workspace/client/src/provider.ts

```diff
@@ -2,6 +2,8 @@ import { init as bootstrap } from "./framework/bootstrapper";
 import { init as initialisePlatform } from "./framework/platform/platform";
 
 window.addEventListener("DOMContentLoaded", async () => {
+	fin.me.interop = fin.Interop.connectSync(fin.me.uuid, {});
```

Comment:
Checked with Michael and although this works the suggestion is to do it after the platform has been initialized.

Reply:
Moved to after initialization